src/components/layout.tsx

```diff
@@ -38,51 +40,117 @@ function Layout({ children }: LayoutProps) {
   `)
   const siteTitle = data.site.siteMetadata.title
   const [toggleNav, setToggleNav] = React.useState(false)
-
+  const open = "bg-black"
   return (
-    <div className={`site-wrapper ${toggleNav ? `site-head-open` : ``}`}>
+    <div className={`${toggleNav ? `px-8` : `px-8`}`}>
       <header className="site-head mb-8">
-        <div className="site-head-container py-6">
+        <div className="site-head-container py-12 pl-2 flex font-merri leading-none">
           <button
-            className="nav-burger"
+            className="flex visible lg:hidden mt-1"
             onClick={() => setToggleNav(!toggleNav)}
           >
-            <div
-              className="hamburger hamburger--collapse"
-              aria-label="Menu"
-              aria-controls="navigation"
+            <svg
```

Comment:
We are mixing 2 things here can you please use 
https://react-svgr.com/playground/

to transform the SVG and use a ternary operator 
https://davidl.fr/pattern

Reply:
Fix